tokenizers/Cargo.toml

```diff
@@ -59,11 +58,18 @@ cached-path = { version = "0.5", optional = true }
 aho-corasick = "0.7"
 paste = "1.0.6"
 proc_macros = { path = "./src/utils/proc_macros" }
+once_cell = "1.8"
+cfg-if = "1"
```

Comment:
I don't think that's needed actually.

You can probably write this as pure rust compilation `#cfg`.